grakn-test/test-integration/src/test/java/ai/grakn/test/rule/EngineContext.java

```diff
@@ -161,7 +161,7 @@ public void before() throws Throwable {
         //
         // When using janus, add -Dgrakn.test-profile=janus
         //
-        // The reason is that the default configuration of Grakn uses the Janus factory while the default
+        // The reason is that the default configuration of Grakn uses the Janus session while the default
```

Comment:
I think this change might have been a mistake

Reply:
Yep